src/Components/CreateImageWizard/formComponents/Oscap.js

```diff
@@ -73,11 +78,24 @@ const ProfileSelector = ({ input, showSelector }) => {
   };
 
   return (
-    <FormGroup
-      isRequired={true}
-      label={'Profile to use for this image'}
-      data-testid="profiles-form-group"
-    >
+    <FormGroup isRequired={true} data-testid="profiles-form-group">
+      {'OpenSCAP profile'}
```

Comment:
This should actually be passed into `<FormGroup>` as a `label` prop, like so: `<FormGroup label={'OpenSCAP profile'} ... />`.

Reply:
if I added the label, the question mark moved under the label

Reply:
In this case you'll need to make the popover button a part of the label, then it should be both on the same line and be styled correctly with the little star for a required field.

So something like this:
```javascript
    <FormGroup
      isRequired={true}
      data-testid="profiles-form-group"
      label={
        <>
          OpenSCAP profile
          <Popover
            maxWidth="30rem"
            position="left"
            bodyContent={
              <TextContent>
                <Text>
                  To run a manual compliance scan in OpenSCAP, download this
                  image.
                </Text>
              </TextContent>
            }
          >
            <Button variant="plain" aria-label="About OpenSCAP" isInline>
              <HelpIcon />
            </Button>
          </Popover>
        </>
      }
    >
```

will result in this:
![image](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/73af655f-2a6a-48eb-8414-af804d4d5306)

You could also add `className="pf-u-p-0"` to the button to make it a bit more compact, but as we have similar spacing in few other places in the Wizard I wouldn't be bothered with it in this PR. :)